dagfactory/generators/airbyte_generator.py

```diff
@@ -0,0 +1,161 @@
+from typing import Any,Dict, Generator, Type
+from airflow.models import BaseOperator
+from airflow.models.base import Base
+from requests.exceptions import RequestException
+from ..dagfactory import DagBuilder
+from airflow.utils.dates import days_ago
+from airflow.providers.airbyte.operators.airbyte import AirbyteTriggerSyncOperator
+from airflow.hooks.base import BaseHook
+import subprocess, requests, json
+
+# IMPORTS FOR TESTING
+import ipdb
+
+
+class AirbyteGeneratorException(Exception):
+    pass
+
+class AirbyteGenerator:
+    def __init__(self, dag_builder, params): 
+        self.dag_builder = dag_builder
+        self.AIRFLOW_OPERATOR_FULL_PATH = AirbyteTriggerSyncOperator.__module__+'.'+AirbyteTriggerSyncOperator.__qualname__
+        self.AIRBYTE_TASK_ID_PREFIX = "AIRBYTE_SYNC_TASK_"
+        self.AIRBYTE_DESTINATION_TABLE_PREFIX = "_AIRBYTE_RAW_"
+        self.DBT_CMD_LIST_SOURCES = "dbt ls --resource-type source"
+
+        try:
+            airbyte_connection_name = params["airflow_connection_id"]
+            airbyte_connection = BaseHook.get_connection(airbyte_connection_name)
+        except KeyError:
+            raise AirbyteGeneratorException("`airflow_connection_id` is missing in DAG's configuration YAML file")
+
+        AIRBYTE_API_URL = f"http://{airbyte_connection.host}:{airbyte_connection.port}/api/v1/"
+        AIRBYTE_API_ENDPOINT_LIST_ENTITY = AIRBYTE_API_URL + "{entity}/list"
+        AIRBYTE_API_ENDPOINT_LIST_WORKSPACES = AIRBYTE_API_ENDPOINT_LIST_ENTITY.format(entity="workspaces")
+        AIRBYTE_API_ENDPOINT_LIST_CONNECTIONS = AIRBYTE_API_ENDPOINT_LIST_ENTITY.format(entity="connections")
+        AIRBYTE_API_ENDPOINT_LIST_DESTINATIONS = AIRBYTE_API_ENDPOINT_LIST_ENTITY.format(entity="destinations")
+
```

Comment:
I wouldn't capitalize these variables, as they are not reusable constants, just variables to be used in this scope.